grazer/src/main/java/edu/asu/diging/grazer/core/graphs/impl/GraphManager.java

```diff
@@ -65,17 +69,37 @@ public void init() {
         transformationNames.add("someone_has_sth_end_date");
         transformationNames.add("sth_has_so_end_date");
         transformationNames.add("someone_has_sth_occur_date");
-        transformationNames.add("sth_has_so_occur_date");
+        transformationNames.add("sth_has_so_occur_date");*/
+        
+        
+       if(folder.exists()) {
+	        	if(folder.isDirectory()) {
+	        		fileNames = folder.listFiles();
+	        		if(fileNames != null && fileNames.length > 0) {
+	        			//fileNames[0] contains a hidden file DS_Store, so starting the for loop with index 1
```

Comment:
you can not assume that is always the case. depends on how the os sorts files, if any other files are added to that folder by the OS or application, etc.

you should filter files by saying how that should look (not exclude specific ones).

Reply:
also, use the filter method from the file object.